.github/workflows/nightly.yml

```diff
@@ -303,3 +303,13 @@ jobs:
           # NOTE: Gradle builder creates MSI file that always uses short version format in file name.
           path: build\dist\${{ needs.git_check.outputs.lse_name }}-${{ needs.git_check.outputs.lse_version_short }}-amd64.msi
           name: ${{ needs.git_check.outputs.base_name }}-amd64.msi
+
+      - name: "Build self-contained archive"
+        run: .\gradlew.bat createWindowsPortableZip -x checkstyleMain -x checkstyleTest
+
+      - name: 'Upload self-contained archive'
```

Comment:
'Upload self-contained archive' -> 'Upload self-contained archive for Windows'

Why here single quotes instead of double quotes, as in the other places?

Reply:
For consistency please yes.

Reply:
Done